test/test_acceptance_tests.py

```diff
@@ -0,0 +1,143 @@
+import io
+import sys
+import unittest
+from unittest.mock import patch
+from src.game import Game
+from test.mocks.mock_message import MockMessage
+
+
+class TestAcceptance(unittest.TestCase):
+    def capture_output(self):
```

Comment:
@Aoxian something like this for extraction?

Reply:
yes, though I don't think the name of this function is all that great.

Reply:
I would have went for something along the lines of `game_playthrough()` and have it return:
```python
    return. captured_output.get_value()
```

Then in your tests:
```python
@patch("builtins.input", side_effect=["1", "1", "1", "2", "3", "4", "5", "6", "7", "n"])
    def test_player_X_wins_the_game(self, mock_input):
    expected_message = MockMessage.declare_winner(self, "X")

    game_output = self.game_playthrough()

    self.assertIn(expected_message, game_output)
```

Reply:
Got it.  I had just created a refactor branch so I will do it there.  I like that approach better.

Reply:
https://github.com/t-keazirian/tic-tac-toe/pull/26